libtelemetry/src/testFull/java/com/mapbox/android/telemetry/MapboxTelemetryTest.java

```diff
@@ -270,39 +257,22 @@ public void checksUserAgentNavigationUi() throws Exception {
     Context mockedContext = mock(Context.class, RETURNS_DEEP_STUBS);
     String aValidAccessToken = "validAccessToken";
     String theNavigationUiAndroidAgent = "mapbox-navigation-ui-android/";
-    MapboxTelemetry theMapboxTelemetry = obtainMapboxTelemetryWith(mockedContext, aValidAccessToken,
-      theNavigationUiAndroidAgent);
+    MapboxTelemetry theMapboxTelemetry = obtainMapboxTelemetryWith(mockedContext, aValidAccessToken);
     theMapboxTelemetry.enable();
-    boolean validRequiredParameters = theMapboxTelemetry.checkRequiredParameters(aValidAccessToken,
-      theNavigationUiAndroidAgent);
+    boolean validRequiredParameters = theMapboxTelemetry.checkRequiredParameters(aValidAccessToken);
     assertTrue(validRequiredParameters);
   }
 
   @Test
   public void checksUserAgentEvents() throws Exception {
     Context mockedContext = mock(Context.class, RETURNS_DEEP_STUBS);
     String aValidAccessToken = "validAccessToken";
-    String theEventsAndroidAgent = "MapboxEventsAndroid/";
-    MapboxTelemetry theMapboxTelemetry = obtainMapboxTelemetryWith(mockedContext, aValidAccessToken,
-      theEventsAndroidAgent);
+    MapboxTelemetry theMapboxTelemetry = obtainMapboxTelemetryWith(mockedContext, aValidAccessToken);
     theMapboxTelemetry.enable();
-    boolean validRequiredParameters = theMapboxTelemetry.checkRequiredParameters(aValidAccessToken,
-      theEventsAndroidAgent);
+    boolean validRequiredParameters = theMapboxTelemetry.checkRequiredParameters(aValidAccessToken);
     assertTrue(validRequiredParameters);
   }
 
-  @Test
-  public void checksNullUserAgent() throws Exception {
```

Comment:
should this be replaced with an `checkEmptyUserAgent()`?

Reply:
No because user agent is no longer passed as a constructor parameter. The user agent that is composed is tested in UserAgentSDKInfoTest.java